WebDevStudios/wd_s
# Feature/tailwind

Closes #545
Closes #546 
Closes #547

### DESCRIPTION ###
- Adds support for Tailwind
- Updates wd_s to strip out opinionated styles and elements
- Removes all ACF JSON/ACF Blocks functionality

### OTHER ###

### STEPS TO VERIFY ###
1. Check out the branch
2. Run `npm i`
3. Off to the races!

### DOCUMENTATION ###
Yes, this is currently updated on my local awaiting for this to be approved before deploying.
